Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute_description.md

```diff
@@ -0,0 +1,4 @@
+This integration provides the ability to import Palo Alto Networks - Prisma Cloud Compute alerts into Demisto.
+
+Important: Before you can use the Prisma Cloud Compute integration on Demisto, there are several configuration steps requeired on the Prisma Cloud Platform. 
+Please follow the detailed instructions in the README.md file to configure the integration properly. 
```

Comment:
Copy here the instructions from the README. Just the part `Configure Demisto alert profile in Prisma Cloud Compute:`